typhos/ui/widgets/positioner.ui

```diff
@@ -467,7 +473,7 @@ Screen</string>
            <string/>
           </property>
           <property name="text">
-           <string>error_label</string>
+           <string/>
```

Comment:
I have no idea what happened here. I'm tempted to revert this specific change manually. I'll spend a moment figuring out what it actually is.

Reply:
Reverted